smart-contracts/contracts/interfaces/ILockCore.sol

```diff
@@ -34,12 +34,19 @@ interface ILockCore {
     payable;
 
   /**
-   * @dev Called by owner to wiwthdraw all funds from the lock.
+   * @dev Called by owner to withdraw all funds from the lock.
    */
   function withdraw(
   )
     external;
 
+  /**
+   * @dev Called by owner to partially withdraw funds from the lock.
+   */
+  function partialWithdraw(uint
```

Comment:
We may want to include a variable name here.  It is option in the interface, but including a variable name offers a bit more data for people viewing the interface, and is consistent with the rest of the interface.

Reply:
yes, good call.

Reply:
done.